docs/reference/query-dsl/match-query.asciidoc

```diff
@@ -261,7 +261,7 @@ GET /_search
 [[query-dsl-match-query-cutoff]]
 ===== Cutoff frequency
 
-deprecated[7.3.0,"This option can be omitted as the <<query-dsl-match-query>> can skip block of documents efficiently, without any configuration, provided that the total number of hits is not tracked."]
+deprecated[7.3.0,"This option can be omitted as the <<query-dsl-match-query>> can skip blocks of documents efficiently, without any configuration, provided that the total number of hits is not tracked."]
```

Comment:
I don't exactly know if "a block of documents" or "blocks of documents" is more precise, but I think the important part is the skipping here. Maybe @matriv knows who seems to have touched that line last, sorry I didn't go down history to see if somebody else put it there).
Also note there is a similar occurance in L302 of this file that we might want to change in a similar way.

Reply:
yep "skip **a** block of documents" also sounds better to me.
I wasn't sure either, "skip block**s** of documents" doesn't sound quite right to me somehow.

Reply:
I don't recall the terminology of a 'block' in Elasticsearch, is this a new ES concept or just common language to indicate more than one?

Reply:
I think its Lucene lingo, the organize documents in block IIRC...

Reply:
@missinglink sorry for dropping the ball on this, I checked with some Lucene folks and the plural "can skip blocks of code" sound better to them. So the PR  can stay like it is. Sorry for the nitpicking, will run tests and merge once they pass